homeassistant/components/wink/alarm_control_panel.py

```diff
@@ -14,11 +16,17 @@
 
 from . import DOMAIN, WinkDevice
 
+_LOGGER = logging.getLogger(__name__)
+
 STATE_ALARM_PRIVACY = "Private"
 
 
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the Wink platform."""
+    _LOGGER.warning(
+        "The Wink integration has been deprecated and will be removed in "
+        "Home Assistant Core 2021.12"
+    )
```

Comment:
This is an integration... why throw a deprecation warning on every platform instead of doing it centrally?

Reply:
Please revert the platform warnings

Reply:
Does that really matter ? this is going to be in for 3 months and then be deleted. Why use more time on this as necessary ?

If it really disturbs you can always to a followup PR.

Reply:
It now spams the logs for existing users and this is not how we deprecate integrations in general.
Please revert the platform warnings.

Reply:
There are 7+ active users, and I tried to ensure they got an informative warning, whenever they started HA, that is in my mind being user friendly.

I implemented the change in schema you demanded, which is an anonymous message leading to confusion.

In Lyft you did what I did, you did not block the schema, but gave the users an informative message….but you deny me the same possibility.

Please lets end this bad discussion, and get the PR merged, so it can be forgotten.

Reply:
> In Lyft you did what I did, you did not block the schema

Lyft is a platform integration, which does not have a generic integration setup.
But you are now making me run into circles.

> Please lets end this bad discussion, and get the PR merged, so it can be forgotten.

?

I'm not sure what you are looking for, but please adjust this PR into how we generally handle this. If not, I'm not going to approve this PR.